### PR TITLE
Prevent mobile browsers from refreshing page on scroll

### PR DESCRIPTION
**Pull request by Kassy2048**
Fix https://github.com/renpy/renpy/issues/4991

It might be needed to set this CSS property to the "html" element as well (but not for Chrome on Android at least).

### CODE REVIEW

**Comment by renpytom** [comment]
Mdn says it should be on html, not body.

**Comment by Kassy2048** [comment]
It did not work when I've added it to the html element only, but it did as soon as I've added it to the body element. Removing it from the html element did not change anything. My guess is it is needed on the html element only when the page is inside an iframe. I can add it to both if you think it makes more sense.

You can test it with the following URLs (scroll down to see if the page refreshes):
* [Original](https://kassy2048.github.io/renpy/refresh-fix/shell-original.html)
* [On html element](https://kassy2048.github.io/renpy/refresh-fix/shell-html.html)
* [On body element](https://kassy2048.github.io/renpy/refresh-fix/shell-body.html)
* [On body and html elements](https://kassy2048.github.io/renpy/refresh-fix/shell-body+html.html)

It only makes a difference with Chrome on Android, as Firefox does not seem to implement such a way to refresh the page.

**Comment by Kassy2048** [comment]
It looks like Safari/Chrome on iOS needs that property set to the html element while Chrome on Android needs it to the body element. Also the property value was wrong because I misread the MDN explanations.

**Comment by renpytom** [comment]
Tl;dr - @Kassy2048 is awesome, and the web sucks as much as ever.
